locales/en/translation.json

```diff
@@ -200,6 +200,7 @@
     "show-dictionary": "Show dictionary",
     "show-book-chapter-navigation": "Show book/chapter navigation",
     "show-header-navigation": "Show header navigation",
+    "show-tab-search": "Show tab search",
```

Comment:
Maybe "Show **current** tab search" is more descriptive.

Reply:
Let's ask a native speaker ... @alerque What do you think?

This is for an option that toggles the search form which allows the user to search within the currently active tab.

Reply:
The expression "show tab search" suggests to me a quest to find that tiny tab you left open in a sea of tabs. I had to check several times to confirm you meant something else because my brain didn't want to break out of that assumption. Given your description "Show current tab search" would be better, or maybe "Show search in current tab" as a more natural reading.

That being said I may be a "native" speaker but having lived in Turkey for closing in on two decades and being married to a native Russian speaker hasn't done wonders for my English. Much public shade has been cast on my native speaker credentials.